views/macros/medical-consent.njk

```diff
@@ -1,5 +1,5 @@
-{% macro medicalConsent(key, values, value, question, errors, attributes) %}
-    <div class="{{ 'has-error' if errors and errors[key] }}">
+{% macro medicalConsent(id, items, selectedValue, question, errors, attributes) %}
```

Comment:
Just a heads-up that I'm planning to junk the medicalConsent macro, since the consent page new design is substantially different than before. (see #118 )